app/build.gradle

```diff
@@ -5,6 +5,7 @@ def appVersionCode = System.getProperty('versionCode', defaultAppVersionCode).to
 
 repositories {
     maven { url 'https://maven.google.com' }
+    maven { url "https://artifactory.us-east-1.aws.aol.com/artifactory/maven-mapquest-us-east-1-local" }
```

Comment:
@akac is this still a valid repo?

Reply:
Not anymore… this needs to change. I'm not yet sure what the actual repo needs to be as it depends what dependency is using it. However we can start with:
   ```maven { url 'https://artifactory.ouroath.com/artifactory/android-local-release' }
    maven { url 'https://artifactory.ouroath.com/artifactory/android-local-snapshot' }
    maven { url 'https://edge.artifactory.ouroath.com:4443/artifactory/remote-repos/' }```